server/ServerApp.cpp

```diff
@@ -1757,14 +1757,14 @@ int ServerApp::AddPlayerIntoGame(const PlayerConnectionPtr& player_connection) {
     }
 
     if (empire_id == ALL_EMPIRES)
-        return false;
```

Comment:
is this change relevant to the subject of the pull request?

Reply:
Reconnection doesn't work properly without this fix.